lib/factory_inspector.rb

```diff
@@ -22,8 +22,8 @@ def self.factory_inspect(factory)
         hash[:type] = "association"
         name_and_traits = attr.factory
         if name_and_traits.class == Array
-          hash[:factory_name] = name_and_traits.shift.to_s
-          hash[:traits] = name_and_traits.map{ |tr| tr.to_s }
+          hash[:factory_name] = name_and_traits.first.to_s
+          hash[:traits] = name_and_traits.drop(1).map{ |tr| tr.to_s }
```

Comment:
Space missing to the left of {.<br>Pass `&:to_s` as an argument to `map` instead of a block.